dolphin/alert_manager/alert_processor.py

```diff
@@ -31,45 +31,20 @@ def __init__(self):
     def process_alert_info(self, alert):
         """Fills alert model using driver manager interface."""
 
-        ctxt = context.get_admin_context()
-
-        # Trap source ip should be configured as part of alert source
-        # configuration. Incoming source ip will be mapped with Trap source
-        # ip in db and storage id will be obtained As config flow not exists
-        # now Currently source ip is mapped with access_info ip address
-
-        # First retrieve access_info from source ip and get storage info
-        # using storage id
-        # This is a temporary mechanism till the alert config flow is handled
-        filters = {'host': alert['transport_address']}
-
-        try:
-            access_info = db.access_info_get_all(ctxt, filters=filters)
-        except Exception:
-            msg = "Access information could not be found with host %s." \
-                  % alert['transport_address']
-            raise exception.AccessInfoNotFound(message=msg)
-
-        # For given source ip, there should be unique access_info
-        if len(access_info) != 1:
-            msg = "Failed to get unique access information with host %s." \
-                  % alert['transport_address']
-            raise exception.InvalidResults(message=msg)
-
         try:
-            storage = db.storage_get(context, access_info[0]['storage_id'])
+            storage = db.storage_get(context, alert['storage_id'])
         except Exception:
```

Comment:
plz handle the exact exception. This way need not to add id=alert['storage_id'] again at line #37

Reply:
Done